api/internal/plugins/loader/loader.go

```diff
@@ -225,25 +229,34 @@ func (l *Loader) loadExecOrGoPlugin(resId resid.ResId) (resmap.Configurable, err
 	if err != nil {
 		return nil, err
 	}
-	// First try to load the plugin as an executable.
-	p := execplugin.NewExecPlugin(absPluginPath)
-	if err = p.ErrIfNotExecutable(); err == nil {
-		return p, nil
-	}
-	if !os.IsNotExist(err) {
-		// The file exists, but something else is wrong,
-		// likely it's not executable.
-		// Assume the user forgot to set the exec bit,
-		// and return an error, rather than adding ".so"
-		// to the name and attempting to load it as a Go
-		// plugin, which will likely fail and result
-		// in an obscure message.
-		return nil, err
-	}
-	// Failing the above, try loading it as a Go plugin.
-	c, err := l.loadGoPlugin(resId, absPluginPath+".so")
-	if err != nil {
-		return nil, err
+	var c resmap.Configurable
+	for i, pluginPath := range absPluginPath {
```

Comment:
In line with my comment above, I don't think we want to do this in a loop here--we only want to load the plugin once, and we should be able to pick a valid location for it from `l.AbsolutePluginPath(resId)` above.